torch/_torch_docs.py

```diff
@@ -5872,6 +5872,39 @@ def merge_dicts(*dicts):
     tensor([ 0,  2,  0,  1])
 """.format(**single_dim_common))
 
+add_docstr(torch.argwhere,
```

Comment:
`torch.nonzero` documentation has a warning about host-device synchronization; we should mention it here, too.

Reply:
Ah. Thanks! Sorry missed that one!